Graphs/Astar.js

```diff
@@ -0,0 +1,107 @@
+/**
+ * Author: Mathang Peddi
+ * A* Search Algorithm implementation in JavaScript
+ * A* Algorithm calculates the minimum cost path between two nodes.
+ * It is used to find the shortest path using heuristics.
+ * It uses graph data structure.
+ */
+
+function createGraph(V, E) {
+  // V - Number of vertices in graph
+  // E - Number of edges in graph (u,v,w)
+  const adjList = [] // Adjacency list
+  for (let i = 0; i < V; i++) {
+    adjList.push([])
+  }
+  for (let i = 0; i < E.length; i++) {
+    adjList[E[i][0]].push([E[i][1], E[i][2]])
+    adjList[E[i][1]].push([E[i][0], E[i][2]])
+  }
+  return adjList
+}
+
+// Heuristic function to estimate the cost to reach the goal
+// You can modify this based on your specific problem, for now, we're using Manhattan distance
```

Comment:
This is not a useful manhattan distance. This is just the absolute distance between vertex IDs. Instead you want some kind of associated "points" (say, in 2d or 3d space) for which you compute distances.

In any case, the heuristic function should probably be a (mandatory) parameter.

Reply:
I have made the changes, I am using Euclidian Distance here, do I need to make any other change here?